Wordle.Api/Wordle.Api/Services/LeaderboardService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Data;
+
+namespace Wordle.Api.Services
+{
+    public class LeaderboardService
+    {
+        private readonly AppDbContext _db;
+
+        public LeaderboardService(AppDbContext db)
+        {
+            _db = db;
+        }
+
+
+        public async Task<IEnumerable<Player>> GetTopPlayers(int? count)
+        {
+            count ??= 10;
+            
+            var players = await _db.Players
+                .Where(player => player.GameCount > 10)
+                .OrderBy(p => p.AverageAttempts)//Lowest average attempts
+                .ThenByDescending(p => p.GameCount)//Then by highest number of games played
+                .Take(count.Value)
+                .ToListAsync();
+            return players;
```

Comment:
If we have average seconds per game data, maybe we can use that as well in here? It depends on how you want to calculate the score.